src/color_laws.jl

```diff
@@ -127,13 +127,87 @@ function cal00_invum(x::Real, Rv::Real)
     if x > 1 / 0.12
         return 0.0x
     elseif x > 1 / 0.63
-        k = 2.659 * (((0.011x - 0.198)x + 1.509)x - 2.156)
+        k = 2.659 * Poly([-2.156, 1.509, -0.198, 0.011])(x)
     elseif x >  1 / 2.2
-        k = 2.659 * (1.040x - 1.857)
+        k = 2.659 * Poly([-1.857, 1.040])(x)
```

Comment:
I realize I initially added the Polynomials here, but now that I think about it we really shouldn't be using them. Instantiating a polynomial requires an allocation, whereas using `@evalpoly x -2.156 1.508 -0.198 0.011`, for example, has no allocations.

Reply:
Updated all to `@evalpoly` except in `ccm89_invum`.